aggregates/build.gradle.kts

```diff
@@ -1,6 +1,14 @@
 plugins {
     kotlin("multiplatform")
+    id("org.jetbrains.compose") version "1.7.1"
     id("com.android.library")
+    alias(libs.plugins.kotlin.compose.compiler)
+}
+
+repositories{
```

Comment:
do you need this?. It isn't added in all repositories {...} in the root gradle?

Reply:
I think it makes sense to include the repositories in the module so that it can be use straightfoward without adding anything else to the main build.gradle